### DISCUSSION TITLE
CSV Delimiter with UQL

### DISCUSSION THREAD

**Discussion by irving-derin-formlabs** [grafana/grafana-infinity-datasource#393] [Q&A] [answered]
I'm trying to parse a tab delimited CSV file from a URL. When using the CSV settings it works exactly as expected and I can change the parsing delimiter super easy! Is there an example or a way to use a tab delimiter with UQL?

**Comment by yesoreyeram** [comment, marked as answer]
See if this helps https://github.com/yesoreyeram/uql/blob/main/src/grammar/tests/grammar.parse.test.ts#L19

Example `parse-csv --delimiter ":" --ignoreError true --headers "foo,bar"`

**Reply by irving-derin-formlabs** [reply]
This worked perfectly! Thank you
